src/gc/impl/precise/gc.d

```diff
@@ -335,6 +374,9 @@ class ConservativeGC : GC
     }
 
 
+    /**
```

Comment:
Didn't we remove these empty comments explicitly in one of the recent commits to the GC?